### PR TITLE
Update the-event-system.md for v2

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

File: core/the-event-system.md [hunk 1]
```diff
@@ -1,10 +1,87 @@
 # The event system
 
-API Platform leverages [events triggered by the Symfony HTTP Kernel](http://symfony.com/doc/current/components/http_kernel/introduction.html#creating-an-event-listener)
-to provide a powerful set of convenient hooks at all times of the object lifecycle.
+API Platform Core implements the [Action-Domain-Responder](https://github.com/pmjones/adr) pattern. This implementation
+is covered in depth in the [Creating custom operations and controllers](operations.md#creating-custom-operations-and-controllers]
+chapter.
+
+Basically, API Platform Core execute an action class that will return an entity or a collection of entity. Then a series
+of event listeners are executed that validate the data, persist it in database, serialize it (typically in a JSON-LD document)
```
Comment (teohhanhui) [inline core/the-event-system.md]: which validate the data, persist it in the database, ...

File: core/the-event-system.md [hunk 2]
```diff
@@ -1,10 +1,87 @@
...
+and create an HTTP response that will be sent to the client.
+
+To do so, API Platform Core leverages [events triggered by the Symfony HTTP Kernel](https://symfony.com/doc/current/reference/events.html#kernel-events).
+You can also hook your own code to those events. They are very handy and powerful extension points available at all times
```
Comment (teohhanhui) [inline core/the-event-system.md]: at all points of the request lifecycle.

File: core/the-event-system.md [hunk 3]
```diff
@@ -1,10 +1,87 @@
...
+            ->setTo('contact@les-tilleuls.coop')
+            ->setBody(sprintf('The book #%d has been added.', $book->getId()));
+
+        $this->mailer->send($message);
+    }
+}
+```
+
+If you use the standard edition of API Platform, creating the previous class is enough. [DunglasActionBundle](https://github.com/dunglas/DunglasActionBundle)
+(installed by default) will automatically register this subscriber as a service and will inject its dependencies using [th
```
Comment (soyuka) [inline core/the-event-system.md]: the typo

**Comment by soyuka** [comment]
:+1:

**Pull request merged**